app/views/learning_partners/index.html.erb

```diff
@@ -12,8 +12,8 @@
     <% @learning_partners.each do |learning_partner| %>
       <li class="my-6">
         <%= link_to learning_partner, class: "link" do %>
-          <div class="flex items-center gap-4 p-2 text-base box-shadow-medium">
-            <%= image_tag partner_logo(learning_partner, :small), alt: "Logo", class: "h-full w-[80px] rounded-l object-cover" %>
+          <div class="flex items-center gap-4 py-2 px-6 text-base box-shadow-medium rounded-lg">
```

Comment:
Whats the logic here ? Why does it work by adding more padding ?

Reply:
padding was missing here before so added the appropriate padding